src/Message/AbstractRequest.php

```diff
@@ -421,6 +421,27 @@ public function setProductReference($value)
         return $this->setParameter('productReference', $value);
     }
 
+    /**
+     * Returns the invoice id, which is normally the first element of invoice numbers array
+     *
+     * @return null|string
+     */
+    public function getInvoiceId()
```

Comment:
this is the gateway's identifier, so i think it should be called getInvoiceReference